.gitignore

```diff
@@ -59,3 +59,5 @@ target/
 
 # Mac OS
 .DS_Store
+
+**.bak
```

Comment:
why **?

Reply:
Sorry femtotrader, could you ignore this pull request please. It is not meant to be public yet, experimental feature. I will share as soon as possible.

Reply:
No problem